Assignments/stu5/assignments.py

```diff
@@ -74,3 +80,23 @@ def hello_world(x):
     for i in range(i):
         print("Hello World from Python!")
 
+def array_to_string(array):
+    string = ''
+    for numbers in array:
```

Comment:
I don't think we need this for loop if we are going to use `.join`.

Reply:
PR UPDATED